provisioning/roles/php/tasks/main.yml

```diff
@@ -1,15 +1,15 @@
-- name: error when trying to use something else than PHP 5.6 on jessie
+- name: error when trying to use something else than PHP 5.6 or PHP 7.0 on jessie
```

Comment:
yes, it is a mess. Thanks for the input, will try to come up with a cleaner solution.